openstack/create.sh

```diff
@@ -1,33 +1,31 @@
 #!/bin/bash
 set -euo pipefail
 
-# gcloud compute resource-policies create instance-schedule "$EIRINI_STATION_USERNAME-shutdown-schedule" \
-#   --description="shut down the machine every day at 19:00 UTC" \
-#   --project="cf-on-k8s-wg" \
-#   --region="europe-west2" \
-#   --vm-stop-schedule="0 19 * * *"
-# gcloud compute instances create "$EIRINI_STATION_USERNAME-eirini-station" \
-#   --project="cf-on-k8s-wg" \
-#   --metadata=ssh-keys="$EIRINI_STATION_USERNAME:$(ssh-add -L)" \
-#   --image-project="ubuntu-os-cloud" \
-#   --image-family="ubuntu-2204-lts" \
-#   --machine-type="e2-custom-8-16384" \
-#   --boot-disk-size="100GB" \
-#   --boot-disk-type="pd-ssd" \
-#   --zone="europe-west2-a" \
-#   --resource-policies="$EIRINI_STATION_USERNAME-shutdown-schedule"
 echo Creating server "$EIRINI_STATION_USERNAME-eirini-station"
 openstack server create "$EIRINI_STATION_USERNAME-eirini-station" \
   --flavor "g_c8_m16" \
-  --image "SAP-compliant-ubuntu-22-04" \
+  --image "SAP-compliant-ubuntu-20-04" \
```

Comment:
Why did you end up using 20-04 instead of 22-04? It would be best to use the latest available option. We are also wondering what is the difference between the sap compliant ubuntu and the standard one and which one would be batter for our use-case.

Reply:
when the issue with the SNAPs came up - i switched back to 20-04. But then I saw that the snap is also disabled there, but forgot to switch back. I'll go with the 22-04 version now